client/src/main/proto/spine/validation.proto

```diff
@@ -61,19 +61,23 @@ extend google.protobuf.FieldOptions {
 
 // A validation constraint violation for a message field.
 message ConstraintViolation {
-    // An interpolated error message for this constraint violation.
+    // An error message for this constraint violation
+    // (or a template which contains `%s` format specifiers).
```

Comment:
If it contains format placeholders, then it's not a `message`, it's `message_format`, or, better, `format`.